### PR TITLE
documentation on FAQs Let’s Encrypt SSL certificates

### PR DESCRIPTION
**Pull request by fakela**
This PR is to fix documentation on FAQs Let’s Encrypt SSL certificates #2444

### CODE REVIEW

**Comment by SueSmith** [comment]
Hi @fakela! Thanks so much for the PRs. 🙌 

So we actually have a docs style guide: https://github.com/postmanlabs/postman-docs/blob/develop/DOCS_STYLE_GUIDE.md

Typically in a case like this we would build the information into the body of the text in the doc rather than adding it as an faq section (in paragraphs and sentences rather than in list style). We also don't use first person in our docs by default ("I" in this case).

Please give me a shout if I can provide any other info! I'll hold off on your other PRs for now but your contribution is much appreciated. 🙂

**Comment by fakela** [comment]
@SueSmith  thanks for your comment and suggestion👍 . Based on your suggestion I came up with this sentence as it covers the answers given in the faq, would it be okay under **using a certificate** section

Note: Certificates are valid for 90 days and are renewed automatically 7 days before it expires.

**Comment by SueSmith** [comment]
Thanks for following up @fakela - few notes, please don't feel obliged to action all this, just to give you a sense of how we edit content in the docs team:

* I think a note in that section would work 👍 
* For notes we normally use markdown blockquotes (`>`) rather than including "note" in the text (see style guide)
* The sentence you have would need "it" changed to "they" because the first part of the sentence is plural, however I'd rework it anyway, as follows:
 * We try not to use passive language so I'd reword to use active phrases, and would split it into a couple of sentences to highlight the automatic part. We also spell out numbers up to ten.. e.g. something like this: "Certificates renew automatically—you do not need to carry out any manual steps. When a certificate is generated it has a 90 day expiry date and will renew seven days before it expires."
* I think we would also need to indicate that this information is specifically for Postman managed Let’s Encrypt SSL certificates since the certificates doc is more general if that makes sense

Again please don't feel obliged to go through all these, for a community contribution we would typically make some of the edits within the team but I know you were interested in how we work on the docs.. Thanks again! 🙂

**Comment by fakela** [comment]
> Thanks for following up @fakela - few notes, please don't feel obliged to action all this, just to give you a sense of how we edit content in the docs team:
> 
> * I think a note in that section would work 👍
> * For notes we normally use markdown blockquotes (`>`) rather than including "note" in the text (see style guide)
> * The sentence you have would need "it" changed to "they" because the first part of the sentence is plural, however, I'd rework it anyway, as follows:
> * We try not to use passive language so I'd reword to use active phrases and would split it into a couple of sentences to highlight the automatic part. We also spell out numbers up to ten.. e.g. something like this: "Certificates renew automatically—you do not need to carry out any manual steps. When a certificate is generated it has a 90 day expiry date and will renew seven days before it expires."
> * I think we would also need to indicate that this information is specifically for Postman managed Let’s Encrypt SSL certificates since the certificates doc is more general if that makes sense
> 
> Again please don't feel obliged to go through all these, for a community contribution we would typically make some of the edits within the team but I know you were interested in how we work on the docs.. Thanks again! 🙂


@SueSmith Thanks for the detailed explanation. I would use this "Certificates renew automatically—you do not need to carry out any manual steps. When a certificate is generated it has a 90 day expiry date and will renew seven days before it expires" to update the docs.

**Comment by fakela** [comment]
@SueSmith Please review

**Review by SueSmith — APPROVED** [review]
moved this and added a little detail..
